backend/src/main/java/dccs/academy/entities/CertificateEntity.java

```diff
@@ -0,0 +1,114 @@
+package dccs.academy.entities;
+
+import jakarta.persistence.*;
+import org.hibernate.annotations.OnDelete;
+import org.hibernate.annotations.OnDeleteAction;
+
+import java.time.LocalDateTime;
+import java.util.List;
+
+
+@Entity
+@Table(name = "certificates",schema = "certificates")
+public class CertificateEntity {
+    public enum CertificateType {
+        PERMISSION_OF_PRINTING,
+        OHSAS_18001,
+    }
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Long id;
+
+    @Column(name = "valid_from")
+    private LocalDateTime validFrom;
+
+    @Column(name = "valid_to")
+    private LocalDateTime validTo;
+
+    @Column(name = "certificate_type",nullable = false)
+    private CertificateType certificateType;
+
+    @Column(name = "file_id",nullable = false)
+    private String pdf_url;
```

Comment:
Name of the field should match name of column, just in camelCase so please rename it.